app/controllers/application_drafts_controller.rb

```diff
@@ -142,10 +144,10 @@ def current_team
   end
 
   def extended_team
-    @extended_team ||= begin
-                        team = ApplicationDraft.find(params[:id]).team
-                        team if (team.coaches + team.mentors).include? current_user
-                      end
+    # @extended_team ||= begin
+    #                     team = ApplicationDraft.find(params[:id]).team
+    #                     team if (team.coaches + team.mentors).include? current_user
+    #                   end
```

Comment:
There's no need for comitting commented code as Git allows you to find it back when needed. If this method is meant to be disabled (e.g. because it's not used) it should just be removed.